lib/jblond/Diff/Renderer/Text/Json.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace jblond\Diff\Renderer\Text;
+
+use jblond\Diff\Renderer\MainRendererAbstract;
+
+/**
+ * Unified diff generator for PHP DiffLib.
+ *
+ * PHP version 7.3 or greater
+ *
+ * @package         jblond\Diff\Renderer\Text
+ * @author          Mario Brandt <leet31337@web.de>
+ * @copyright   (c) 2023 Mario Brandt
+ * @license         New BSD License http://www.opensource.org/licenses/bsd-license.php
+ * @version         2.4.0
+ * @link            https://github.com/JBlond/php-diff
+ */
+
+/**
+ * Class Diff_Renderer_Text_Json
+ */
+class Json extends MainRendererAbstract
+{
+    /**
+     * @var array   Associative array containing the default options available for this renderer and their default
+     *              value.
+     */
+    private $subOptions = [
+        'json' => JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE
```

Comment:
'ext-json' in composer.json is specified in 'suggest' and not in 'require' thus may be unavailable.
Perhaps keep it in 'suggest' and when using the json renderer check if the extension is loaded.

Reply:
Done